lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -137,12 +145,12 @@
 
     dns_registration:
         description:
-            - If true web app hostname is not registered with DNS on creation.
+            - Whether or not the web app hostname is registered with DNS on creation. Set to C(false) to register.
```

Comment:
Are you really sure this is what the code does? The name of the parameter sounds like it should work the opposite way. `dns_registration=true` should register and `dns_registration=false` should skip registering. 

The way it's currently documented is counter-intuitive. If this is really how the code works, maybe we need to add a warning of some kind. As a user, I would definitely use this incorrectly, if it works as described here.

Reply:
@MyronFanQiu Can not find dns_registration in azure-python-sdk and Azure API under what conditions it will be registered. It is also impossible to get DNS registration status through YAML. Could you please help to see if this doc description is correct? Thank you very much!

Reply:
@yungezz Could you please enumerate the instructions for dns_registration's parameter? There is no instructions for using dns_registration in Azure API and SDK ! Thank you very much!